### DISCUSSION TITLE
Auto-mount of remote folder fails

### DISCUSSION THREAD

**Discussion by bluegrover** [bb-qq/r8152#421] [Q&A] [answered]
Hello r8152 community!
I have just successfully installed 2.18.1-1 (DSM 7.2) on a DS 720+ (gemini lake) for a UGREEN USB 2.5G USB-A adapter and the adapter is working fine when the DS is booted.
However, what constantly fails is the auto-mount of remote folders at boot-up. 
I have set up a few folders to be CIFS-mounted from another DS at boot time, which worked and works reliably using the 1GBit LAN connnection (re-tested again with the r8152 driver being disabled). These folders need to be there at boot time because a SW/service I'm running on the 720+ needs these.
With the 2.5GB connection however auto mounting these remote folders always fails.
Any similar experiences or a potential solution for this behaviour?
Thanks and BR, B.

**Comment by bb-qq** [comment, marked as answer]
This driver is loaded at the end of the system boot sequence for safety reasons. The order is later than the file system mount.

To address this issue, you can disable the automatic fstab mount and use the task scheduler to mount the file system after system startup.

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
